lib/constants.js

```diff
@@ -13,6 +13,7 @@ module.exports = {
                 'privacy_consent',
                 'address',
                 'about_me'
+                //does gsuite_id go here?
```

Comment:
Don't think so, but depends on your implementation of assigning gsuite_id to users on creation.